qiskit/scheduler/lowering.py

```diff
@@ -59,20 +60,49 @@ def lower_gates(circuit: QuantumCircuit, schedule_config: ScheduleConfig) -> Lis
 
     def get_measure_schedule() -> CircuitPulseDef:
         """Create a schedule to measure the qubits queued for measuring."""
-        sched = measure(qubits=list(qubit_mem_slots.keys()),
-                        inst_map=inst_map,
-                        meas_map=schedule_config.meas_map,
-                        qubit_mem_slots=qubit_mem_slots)
+        sched = Schedule()
+        # Exclude acquisition on these qubits, since they are handled by the user calibrations
+        acquire_excludes = {}
+        if Measure().name in circuit.calibrations.keys():
+            qubits = tuple(sorted([*qubit_mem_slots]))
```

Comment:
Yes, line 113

Reply:
you can also pass the `qubit_mem_slots` as argument at line 104 because this is mutable. I think this logic is too large to keep it as a nested function.

Reply:
good point